pages/ime.vue

```diff
@@ -1,9 +1,208 @@
+                    <a href="https://www.facebook.com/aaamat.ime" class="link"
+                      >aaamat.ime</a
+                    >
+                  </td>
+                  <td class="nome text-center text-md-left">
+                    <img
+                      class="icon"
+                      width="50px"
+                      src="~/assets/images/instagram.svg"
+                    />
+                    <a
+                      href="https://www.instagram.com/aaamat_imeusp/"
+                      class="link"
+                      >aaamat_imeusp</a
+                    >
+                  </td>
+                </tr>
+              </table>
+            </span>
+          </v-card>
+        </v-col>
+        <v-col
+          cols="12"
+          md="6"
+          class="d-flex justify-start justify-center justify-md-center align-center"
+        >
+          <AppCarrossel :largura="600" :links="fotosAtletica" />
+        </v-col>
+      </v-row>
+      <v-row>
+        <v-col
+          cols="12"
+          md="6"
+          class="d-flex text-center text-md-right justify-md-center justify-center align-center"
+        >
+          <AppCarrossel :largura="500" :altura="400" :links="fotosIme" />
+        </v-col>
+        <v-col
+          cols="12"
+          md="6"
+          class="d-flex justify-center justify-md-center align-center"
+        >
+          <div class="imeDescContainer">
+            <p>
+              O Instituto de Matemática e Estatística é uma unidade de ensino,
+              pesquisa e extensão da Universidade de São Paulo. Possui cursos de
+              graduação e pós-graduação nas áreas de matemática, matemática
+              aplicada, estatística e ciência da computação. Além disso, com
+              corpo docente e discente qualificado, realiza pesquisas
+              científicas em suas áreas-chave e áreas relacionadas, como
+              bioinformática, neuromatemática e educação em matemática. Tem como
+              atividades de extensão, abertas à comunidade externa: cursos de
+              aperfeiçoamento e oficinas para professores da rede pública,
+              cursos de matemática e de computação durante o período de recesso
+              do calendário letivo de graduação, assessoria de estatística e de
+              software livre para pesquisadores, empresas e órgãos públicos.
+              Também promove exposições de objetos matemáticos com o objetivo de
+              propagar conceitos matemáticos diversos à comunidade em geral,
+              além de motivar e ilustrar conteúdos ministrados em sala de aula.
+            </p>
+            <p class="text-md-left text-center">
+              <v-btn
+                color="#142C68"
+                class="white--text mt-5"
+                href="http://ime.usp.br"
+              >
+                Visite o site do IME
+              </v-btn>
+            </p>
+          </div>
+        </v-col>
+      </v-row>
```

Comment:
Sugestão: Abstrair esse pedaço de código para um componente usando template e slot (pq esse mesmo pedaço de código é usado em outras páginas tbm)

Reply:
Fiz isso e defini duas barras comuns das páginas, a primária, que é naquele estilo com o _carousel_ e o _card_, e a secundária, que é formada pela descrição e o _carousel_. A gente pode, inclusive, alterar a _index_ com a barra primária mais tarde, se for necessário, porque o estilo é o mesmo.